astropy/coordinates/matching.py

```diff
@@ -213,7 +213,8 @@ def search_around_3d(coords1, coords2, distlimit, storekdtree="kdtree_3d"):
         The second set of coordinates, which will be searched for matches from
         ``coords1`` within ``seplimit``.  Cannot be a scalar coordinate.
     distlimit : `~astropy.units.Quantity` ['length']
-        The physical radius to search within.
+        The physical radius to search within. It can be a scalar that will be applied
```

Comment:
I'd write "... within. Should broadcast with ``coords1``." Keeps it on one line, and anybody using numpy should be familiar with the term.

Same for the occurrences later on.